src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/HistoryTabPanel.kt

```diff
@@ -0,0 +1,29 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions
+import java.time.Duration
+
+class HistoryTabPanel(
+    private val driver: WebDriver
+) {
+    private val loadMoreEntriesLocator = By.cssSelector("button.show-more-changehistory-tabpanel")
+
+    fun loadAllHistoryEntries(): HistoryTabPanel {
+        if (driver.isElementPresent(loadMoreEntriesLocator)) {
+            val loadMoreButton = driver.findElement(loadMoreEntriesLocator)
```

Comment:
BTW. You can `val loadMoreButton = driver.findElements(loadMoreEntriesLocator).singleOrNull()` and work with a nullable `loadMoreButton: WebElement?` rather than an `isElementPresent`. I believe that's true for vast majority of `isElementPresent` calls.